go-controller/pkg/ovn/ovn_test.go

```diff
@@ -41,10 +46,12 @@ func NewFakeOVN(fexec *ovntest.FakeExec) *FakeOVN {
 	}
 }
 
-func (o *FakeOVN) start(ctx *cli.Context, objects ...runtime.Object) {
+func (o *FakeOVN) start(ctx *cli.Context, egressObjects []runtime.Object, objects ...runtime.Object) {
```

Comment:
nit: instead of passing the egressObjects as an argument (mostly `nil`) everywhere, you can skip the parameter altogether and only in tests where you truly care about fakeEgressClient, do the setting of fakeEgressClient in the  `BeforeEach` section of those specific tests. This is fine for unit-test code.

Reply:
Yeah, this would be a nice reduction in the patch LoC changed. +1

Reply:
We do this for multicast, for example (I think).

Reply:
done